### ISSUE TITLE
Publish usage howto

### ISSUE THREAD

**Issue by pdvyas** [MPJHorner/ERPNext_PHP_API_Wrapper#1]
Hey, 
Good job :)
If you or someone with PHP experience can push a howto / Readme, we can link this repo at https://github.com/frappe/frappe-client (this one is for Python).

**Comment by MPJHorner** [comment]
Sure, I will work on this. Shouldn't take long!

**Comment by pdvyas** [comment]
Awesome!

**Comment by markelayan** [comment]
@MPJHorner  any updates on the howto?

**Comment by MPJHorner** [comment]
@ramielian nope, haven't got the time. By all means feel free to push something, it's quite self explanatory tbh... its a simple wrapper.
